scripts/check-translations.sh

```diff
@@ -0,0 +1,115 @@
+# This script finds `lang.t` or similar translation IDs that are missing in language
+# resources. Note, this script currently **does not** support interpolated
+# translations such as `lang.t(`my.\${variable}.based.${path}`).
+#
+# The script can be run with
+# `sh ./scripts/check-translations.sh ./src './src/languages/*.json' lang.t`
```

Comment:
lets add a yarn command for this, eventually I want to run this on CI and fail builds if we're missing something

Reply:
FYI I needed to `brew install ggrep` 
when I ran it just as is before installing `ggrep` I still saw ✅ messages in the console

<img width="1393" alt="Screen Shot 2022-01-09 at 10 02 46 AM" src="https://user-images.githubusercontent.com/1285228/148692547-c2dba41e-ae7e-4ede-81bc-42a6fc75596f.png">

Reply:
I added the yarn command and updated the script to check for the required commands first. Thanks!